src/main/java/seedu/address/storage/AppointmentArchive.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.storage;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.model.ReadOnlyAppointmentBook;
+
+/**
+ * Represents an archive storage for {@link seedu.address.model.AppointmentBook}.
+ */
+public interface AppointmentArchive {
+
+    Path getArchiveDirectoryPath();
+
+    /**
+     * Archive and remove past appointments from the {@code appointmentBook}.
```

Comment:
Should `Archive and remove` be `Archives and removes`?

Reply:
Yep, I missed that out! Good catch 👍